controller.go

```diff
@@ -686,13 +724,18 @@ func (c *Controller) handleObject(obj interface{}) {
 	}
 }
 
-func makeUserdata(authToken string) string {
+func makeUserdata(authToken string, useSudo bool) string {
 	controlPort := fmt.Sprintf("%d", inletsControlPort)
 
+	var sudo = ""
+	if useSudo {
+		sudo = "sudo"
+	}
+
 	return `#!/bin/bash
 export INLETSTOKEN="` + authToken + `"
 export CONTROLPORT="` + controlPort + `"
-curl -sLS https://get.inlets.dev | sudo sh
+curl -sLS https://get.inlets.dev | ` + sudo + ` sh
```

Comment:
We actually don't need `sudo` at all for cloud-init because it runs as root anyway. I think we can delete this.